azurelinuxagent/ga/env.py

```diff
@@ -98,6 +98,7 @@ def monitor(self):
         If dhcp client process re-start has occurred, reset routes.
         Purge unnecessary files from disk cache.
         """
+        self.protocol_util = get_protocol_util()
```

Comment:
let's add a comment of why this is initialized here

Reply:
Done